examples/torch/common/models/segmentation/icnet.py

```diff
@@ -168,14 +168,23 @@ def __init__(self, input_size_hw, bin_dimensions=None, mode='sum'):
         for dim in self.bin_dimensions:
             pad_h = (dim - (input_size_hw[0] % dim)) % dim
             pad_w = (dim - (input_size_hw[1] % dim)) % dim
-            self.paddings[dim] = [0, pad_w, 0, pad_h]
+            self.paddings[dim] = (0, pad_w, 0, pad_h)
 
     def forward(self, inputs):
         x = inputs.clone()
 
         for dim in self.bin_dimensions:
-            padded_input = F.pad(inputs, self.paddings[dim], mode='constant', value=0)
```

Comment:
why did you remove pad op?

Reply:
`F.pad` is one operation that is incorrectly implemented in torch itself in terms of exporting with shape inference - it produces invalid shape data for the output tensor which then cannot be treated correctly in avg_pool_2d. I originally removed this because this allowed to export FP32 ICNet without the `ATEN_FALLBACK`, but then I hit the proble with INT8 ICNet because FQs don't specify shape inference too.
Anyway, it seems that the `avg_pool_2d` op does the same padding implicitly - I can claim this due to the fact that with and without padding the result accuracy metrics are equivalent.